man/systemd.exec.xml

```diff
@@ -1392,6 +1393,21 @@ RestrictNamespaces=~cgroup net</programlisting>
         that actually require them. Defaults to off.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestrictSUIDSGID=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If set, any attempts to set the set-user-ID (SUID) or
+        set-group-ID (SGID) bits on files or directories will be denied (for details on these bits see
+        <citerefentry><refentrytitle>inode</refentrytitle><manvolnum>7</manvolnum></citerefentry>). If
```

Comment:
This needs a `project=...` attribute so the href is not broken.

Reply:
will fix